src/register-org/components/organisation-type/organisation-type.component.ts

```diff
@@ -47,6 +47,8 @@ export class OrganisationTypeComponent extends RegisterComponent implements OnIn
     }
 
     this.subscription = this.lovRefDataService.getListOfValues(this.CATEGORY_ORGANISATION_TYPE, true).subscribe((orgTypes) => {
+      // To set Other option as the last option
+      orgTypes.push(orgTypes.splice(orgTypes.findIndex((e) => e.key === 'OTHER'), 1)[0]);
```

Comment:
Can you please rewrite the logic.

1. Create a new function that takes old Array and returns an old Array
2. Copy all elements from old array to new array except OTHER
3. If there is other then have the last element as OTHER

Reply:
Moved the logic to app util